src/Core/Error.js

```diff
@@ -35,7 +35,8 @@ function errorException (element, error, from) {
 	}
 
 	return defineProperties(error, {
-		report: {value: report + '\n` from "' + from + '"\n\n' + error.stack + '\n\n', writable: true}
+		report: {value: report + '\n` from "' + from + '"\n\n' + error.stack + '\n\n', writable: true},
+		error: {value: error}
```

Comment:
Why?

Reply:
Solely for connivence, to maintain a reference to the error object when destructuring.

```js
componentDidCatch({error, stack, message, report}) {
fetch(`/report?message=${message}...`)

return error
}
```

Reply:
Why `return error`?

Reply:
The return signature is used as the recovery element.

Reply:
Why would you want to render the error? Is that all this give you?

Reply:
Yes it gives you access to the `Error` object when destructuring. Since `{message, ...error}` would give you an empty object because the properties on the error object are not enumerable.  But yes that's all.

Here's an example of how Error objects are rendered [https://jsbin.com/gopasiduye/edit?js,output](https://jsbin.com/gopasiduye/edit?js,output)